wrapanapi/systems/redfish.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+"""Backend management system classes
+Used to communicate with providers without using CFME facilities
+"""
+from __future__ import absolute_import
+
+from wrapanapi.systems.base import System
+
+
+class RedfishSystem(System):
+    """Client to Redfish API
+    Args:
+        hostname: The hostname of the system.
+        username: The username to connect with.
+        password: The password to connect with.
+    """
+    _api = None
+
+    _stats_available = {
+        'num_server': lambda self: 1,
+    }
+
+    def __init__(self, hostname, username, password, protocol="https", port=None, **kwargs):
+        super(RedfishSystem, self).__init__(kwargs)
+        self.port = port or kwargs.get('api_port', 443)
```

Comment:
Do we need both `port` and `kwargs['api_port']`? I'd simply make `port` positional argument and not expect it in kwargs:

```python
def __init__(self, hostname, username, password, port, protocol="https", **kwargs):
```